recipes/seqan-library/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "seqan-library" %}
+{% set version = "2.2.0" %}
+{% set sha256 = "b5c036a3d2fc2fe5f2d57dcd4d7c523a6df146ab7b44bc789f42004b058d72fd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: https://github.com/seqan/seqan/releases/download/seqan-v{{ version }}/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: cp -r include share $PREFIX
```

Comment:
Shouldn't we building with [CMake]( https://github.com/seqan/seqan/blob/seqan-v2.2.0/CMakeLists.txt )?